strictdoc/core/traceability_index.py

```diff
@@ -537,6 +541,61 @@ def parent_cycle_traverse_(node_id: str) -> List[Any]:
                 datetime.datetime.today(),
             )
 
+    def update_node_mid(self, node: SDocNode, new_mid: str) -> None:
```

Comment:
Just a question, not blocking the review: Is this something that can / should be reused when a MID is changed through server / UI?

Reply:
It is painful to think about this but supporting the MID/HASH autogen in the UI is its own non-trivial epic.